docs/manual/input.md

```diff
@@ -701,6 +701,41 @@ Below is an example of an input_file for the depth average rule:
       output_variable: average_salinity
 ```
 
+### Filter extremes rule
+
+```
+FORMAT
+- filter_extremes_rule:
+      name: <name_of_rule_in_text>
+      description: <description_of_rule_in_text>
+      input_variable: <one_input_variable_name>
+      output_variable: <one_output_variable_name>
+      extreme_type: troughs or peaks
+      distance: <int_of_time_scale>
+      time_scale: second, hour, day, month or year
+      mask: <boolean>
+```
+
+
+The filter extremes rule allows for a filtering of extreme peaks in a dataset (over time). The input variable can be any dimension, as long as it has a time dimension. If the variable mask = True, the output is a variable with the same shape as the input, but only values where the peaks occur and NaN values where no peak occur. If mask = False the output is a samesized variable with 1 (True) at the peak values and NaN elsewhere. Furthermore the user can add a distance (with timescale) as input to define the minimum distance between two peaks/troughs. This mask can be applied to another layer with the combine rule (operation: multiply).
```

Comment:
allows for a filtering? -> filters the extremes in a dataset
extreme peaks is a pleonasm.

Reply:
-> but contains only values at the locations of the peaks, and NaN values elsewhere.
with 1 (True) at the peak -> 1 is not equal to True. So it's not a true mask, since it does not return False
why do we talk about peaks or troughs? Why not max/min?

Reply:
pleonasm is gone
mask = True and mask = False were switched around. woops.
It is not  a True/False array no, but using a True/NaN array it can be easily combined with other layers. Otherwise the none-peak values would all be 0 and that is not what is needed here.... 

There is no 1 max or 1 min, they are peaks and troughs in the dataset. Therefore also the name in the scipy function find_peaks.

Reply:
I know, I was talking about local max/local min (which is also in the scipy documentation). I'll add this to our documentation.